auth/oidc.go

```diff
@@ -46,7 +46,11 @@ const (
 	discoveryConfigPath = "/.well-known/openid-configuration"
 )
 
-var OIDCDiscoveryRetryWait = 500 * time.Millisecond
+var (
+	OIDCDiscoveryRetryWait = 500 * time.Millisecond
+
+	providerConfigSyncInterval = time.NewTicker(1 * time.Second)
```

Comment:
Refreshing each provider once per second seems SUPER aggressive...

For comparison, the old library had a lower bound of 1 minute, and an upper bound of 24 hours, and used them like this:

https://github.com/coreos/go-oidc/blob/e860bd55bfa7d7cb35d30d26a167982584f616b0/oidc/provider.go#L591-L604